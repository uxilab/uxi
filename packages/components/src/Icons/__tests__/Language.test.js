import React from 'react';
import Language from '../Language';

describe('Icon : <Language />', () => {
  it('match snapshot', () => {
    expect(shallow(<Language />)).toMatchSnapshot();
  });
});

