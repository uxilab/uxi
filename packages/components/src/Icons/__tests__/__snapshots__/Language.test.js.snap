// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Language /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 21.8 19.66"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M18.59,13.42,16.16,6.28a.54.54,0,0,0-1,0l-2.57,7.14a.65.65,0,0,0,.37.69h.15a.47.47,0,0,0,.51-.36l.51-1.25a.07.07,0,0,1,.05,0h2.92a.07.07,0,0,1,.05,0l.43,1.22a.65.65,0,0,0,.6.41h.14a.61.61,0,0,0,.32-.33A.47.47,0,0,0,18.59,13.42ZM16.75,11.3s0,0,0,0a.06.06,0,0,1-.05,0H14.57s0,0,0,0a.07.07,0,0,1,0,0L15.6,8.24a.06.06,0,0,1,.1,0Z"
        />
        <path
          d="M8.33,6.3l0,.12V6.3l-.06,0L7,6.29l0,0a0,0,0,0,1,0,0l1.69-3.5a.5.5,0,0,0,0-.38.49.49,0,0,0-.25-.28.53.53,0,0,0-.37,0,.49.49,0,0,0-.28.25L6.26,5.45l0,0h0l0,0L4.65,2.3A.51.51,0,0,0,4.21,2a.47.47,0,0,0-.49.47.45.45,0,0,0,0,.23l1.69,3.5a.06.06,0,0,1,0,.06l0,0H4.21a.53.53,0,0,0-.35.14.54.54,0,0,0-.15.35.49.49,0,0,0,.14.34.51.51,0,0,0,.36.15H5.67a.06.06,0,0,1,.06.05V9.76a.52.52,0,0,0,.14.36.48.48,0,0,0,.34.14.49.49,0,0,0,.5-.49V7.32a0,0,0,0,1,0-.05H8.21a.41.41,0,0,0,.36-.14.45.45,0,0,0,.15-.34A.49.49,0,0,0,8.33,6.3Z"
        />
        <path
          d="M20,3.82H12.43v-2A1.81,1.81,0,0,0,10.62,0H1.81A1.81,1.81,0,0,0,0,1.81v8.66a1.82,1.82,0,0,0,1.81,1.81H9.37V14.6a1.68,1.68,0,0,0,1.81,1.5H20a1.68,1.68,0,0,0,1.81-1.5V5.33A1.68,1.68,0,0,0,20,3.82ZM1,10.47V1.81A.81.81,0,0,1,1.81,1h8.81a.81.81,0,0,1,.81.81v8.66a.81.81,0,0,1-.81.81H1.81A.81.81,0,0,1,1,10.47ZM20.8,14.6c0,.23-.34.5-.81.5H11.18c-.46,0-.81-.27-.81-.5V12.28h.25a1.82,1.82,0,0,0,1.81-1.81V4.82H20c.47,0,.81.27.81.51Z"
        />
        <path
          d="M15.53,17.24H13.19l.9.91A4.88,4.88,0,0,1,6.59,14a.38.38,0,0,0-.75,0,5.66,5.66,0,0,0,5.65,5.65,5.57,5.57,0,0,0,3.14-1l.9.9Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
